src/extensions/default/bramble/htmlContent/console.html

```diff
@@ -0,0 +1,20 @@
+<div id="editor-console" class="bottom-panel vert-resizable top-resizer">
+    <div class="toolbar simple-toolbar-layout mainToolbar">
+        <span class="title">Console</span>
```

Comment:
Let's also remember that the strings in this HTML are also going to need to get inserted via localization.